### DISCUSSION TITLE
Add support for A5-04-03 [DONE]

### DISCUSSION THREAD

**Discussion by daschmidt1994** [mak-gitdev/HA_enoceanmqtt#4] [Request] [answered]
Hi,

please add support for A5-04-03

**Comment by mak-gitdev** [comment]
Hello,

Initial support for A5-04-03 has just been pushed.
You should now see that an update is available for the addon. Please stop the addon, then update or reinstall.
Add your A5-04-03 device in the device file then restart the addon.
After that you should see entities corresponding to your device in HA.

Please then report whether the created entities are OK or require modifications.

**Comment by daschmidt1994** [comment]
looks good, just a little too many decimal places
![IMG_20221116_061113](https://user-images.githubusercontent.com/53835578/202089344-998e1f57-5fb8-443f-85c3-f7269e1c39b6.jpg)

**Comment by mak-gitdev** [comment]
Thanks for testing. Great that it works.
I'm wondering if I should manage the rounding or leave it up to each person to do their rounding.
For now I leave it like that.

**Comment by mak-gitdev** [comment, marked as answer]
Hi @daschmidt1994,

I have an update available for A5-04-03.
I added new entities :).
Please see #12 for more details (for A5-04-03, there is no status entity).
